action.yml

```diff
@@ -43,6 +43,9 @@ inputs:
   upload:
     required: false
     description: "Upload analysis results to Codacy"
+  upload-batch-size:
+    required: false
+    description: "Size of result batches to be uploaded to Codacy"
```

Comment:
We should mention the unit for the batch size. I'm guessing it's bytes? The Codacy Analysis CLI [doesn't mention it](https://github.com/codacy/codacy-analysis-cli#commands-and-configuration) either.